pom.xml

```diff
@@ -1597,7 +1600,7 @@
                 <executions>
                     <execution>
                         <goals>
-                            <goal>check</goal>
+                            <goal>apply</goal>
```

Comment:
How to use this plugin? `mvn spotless:apply`?

Is `mvn spotless:apply` command still in effect?

Reply:
When compiling, it will be executed `apply` by default, avoid checking first and then applying.

`mvn spotless:check` and `mvn spotless:apply` are still available.